mne_connectivity/vector_ar/tests/test_var.py

```diff
@@ -29,21 +47,25 @@ def create_noisy_data(
         The number of samples.
     return_A : bool
         Whether to return the A matrix
+    random_state : int
+        The random state to set.
 
     Returns
     -------
-    sample_data : np.ndarray
+    sample_data : np.ndarray (n_channels, n_samples)
```

Comment:
```suggestion
    sample_data : ndarray, shape (n_channels, n_samples)
```
check how mne does it and be consistent.